packages/gatsby-theme-docs/src/components/side-by-side.js

```diff
@@ -33,9 +38,20 @@ const SideBySideContainer = styled.div`
 
 const SideBySide = (props) => {
   const pageData = usePageData();
+  const siteData = useSiteData();
 
   if (pageData.allowWideContentLayout)
-    return <SideBySideContainer>{props.children}</SideBySideContainer>;
+    return (
+      <SideBySideContainer
+        stickyMargin={
```

Comment:
bare minimum margin. Set to 0 when there's no global notification banner, otherwise it's set exactly to the notification height (including padding). We could decide to remove this logic and just add a fixed higher margin (like the example in Stripe does)

Reply:
I suggest a top margin, definitely in the case without the top banner but also a little more when used with the top banner.  Here's an example from the docs kit docs:

<img width="1525" alt="image" src="https://user-images.githubusercontent.com/4006139/205742642-50c380e0-e170-40fd-8940-a9fb1dce8768.png">

Given that the primary use case is in the API docs I think  the `1rem`  padding that the code example has towards the grey background area anyways would be one logical choice.  In other contexts in body content, 0.5rem is a common padding too depending on the situation but I think it's visually better to have a bit more spacing in doubt. 

@Shecki opinons?  That was a long standing feature wish from you, so you have that last word :-) .  See this page and make the screen very wide: 
https://commercetools-docs-84dfah1in-commercetools.vercel.app/api-docs-smoke-test/layouts/endpoint-large-desktop

Reply:
`1rem` looks good to me, in both the situations @nkuehn .
if you agree we can go ahead with this spacing and then fine tune it if needed